windows/RNSVG/PathView.cpp

```diff
@@ -0,0 +1,295 @@
+#include "pch.h"
+#include "PathView.h"
+#if __has_include("PathView.g.cpp")
+#include "PathView.g.cpp"
+#endif
+
+#include <winrt/Microsoft.Graphics.Canvas.Svg.h>
+
+#include "JSValueXaml.h"
+#include "Utils.h"
+
+using namespace winrt;
+using namespace Microsoft::Graphics::Canvas;
+using namespace Microsoft::ReactNative;
+
+namespace winrt::RNSVG::implementation {
+void PathView::UpdateProperties(IJSValueReader const &reader, bool forceUpdate, bool invalidate) {
+  const JSValueObject &propertyMap{JSValue::ReadObjectFrom(reader)};
+
+  for (auto const &pair : propertyMap) {
+    auto const &propertyName{pair.first};
+    auto const &propertyValue{pair.second};
+
+    if (propertyName == "d") {
+      m_commands.clear();
+      m_segmentData.clear();
+
+      if (propertyValue.IsNull()) {
+        m_d.clear();
+      } else {
+        m_d = propertyValue.AsString();
+        ParsePath();
+      }
+    } 
+  }
+
+  __super::UpdateProperties(reader, forceUpdate, invalidate);
+}
+
+void PathView::CreateGeometry(UI::Xaml::CanvasControl const &canvas) {
+  auto const &resourceCreator{canvas.try_as<ICanvasResourceCreator>()};
+  Svg::CanvasSvgDocument doc{resourceCreator};
+  auto const &path{doc.CreatePathAttribute(m_segmentData, m_commands)};
+  Geometry(path.CreatePathGeometry());
+}
+
+void PathView::ParsePath() {
+  char prev_cmd = ' ';
```

Comment:
Is there a longer plan of having this parsing supported by Graphics.Canvas.Svg?  Having to do this parsing here feels rough

Reply:
While investigating how to implement this I came upon [this](https://github.com/microsoft/Win2D/issues/186), but I haven't reached out to the Win2D to see if they have changed their minds. 

This parse code / solution is taken from the [Android implementation](https://github.com/react-native-svg/react-native-svg/blob/develop/android/src/main/java/com/horcrux/svg/PathParser.java), so we at least have parity there.

Reply:
I am hoping you can delete all of the parsing code and just use XamlBindingHelper instead